driver/ppm_fillers.c

```diff
@@ -392,6 +394,20 @@ const struct ppm_event_entry g_ppm_events[PPM_EVENT_MAX] = {
 	[PPME_PAGE_FAULT_E] = {f_sys_pagefault_e},
 	[PPME_PAGE_FAULT_X] = {f_sys_empty},
 #endif
+	[PPME_SYSCALL_CHOWN_E] = {PPM_AUTOFILL, 3, APT_REG, {{0}, {1}, {2} } },
```

Comment:
Am I mistaken or you put all arguments in the *enter* event and not in the exit event, as stated in the PR description?

Reply:
@ret2libc when I submitted this PR I put all the arguments in the exit event but after some comments from @SteveHallNNT here #986, if the arguments are `fd` related, the `fd` information was not saved so I moved the arguments to the enter event.

Reply:
However this means that in some cases you may not be able to retrieve the `pathname` argument because in the enter event accesses to user memory could fail.

Reply:
So what do you recommend?
Should I put the `pathname` in the exit event?

Reply:
Input parameters should go in the enter event, output parameters + userspace pointers in the exit event.

Reply:
Just to make it clear, move only the `pathname` from all the syscalls to the exit event?

Reply:
One more thing, when using `fchmodat` or `fchownat` you can use the special value `AT_FDCWD` for `dirfd`, sysdig list this as an error `ENETDOWN`:
```
2233611 18:06:45.251496870 2 tests (13047) > fchmodat dirfd=-100(ENETDOWN) mode=04 flags=0 
```
Can we do something regarding this?